tests/02-perf-gce.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+
+source "./helpers.bash"
+
+set -e
+
+TEST_NET="cilium"
+NETPERF_IMAGE="tgraf/nettools"
+TEST_TIME=30
+SERVER_LABEL="io.cilium.server"
+CLIENT_LABEL="io.cilium.client"
+SERVER_NAME="server"
+CLIENT_NAME="client"
+
+# Only run these tests if BENCHMARK=1 has been set
+if [ -z ${BENCHMARK} ] && [ -z ${GCE} ]; then
```

Comment:
You want to abort if `BENCHMARK` or `GCE` is a zero string. I think you really mean `||`